gslib/commands/iam.py

```diff
@@ -309,6 +316,55 @@ class IamCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    sub_command = self.args.pop(0)
```

Comment:
Not familiar with how sub-args are parsed by the shim, but it looks like the idea is to mutate `self.args` here so that `get_gcloud_storage_args` doesn't see the sub command?

Reply:
Yes!